doc/innovation_roadmap.rst

```diff
@@ -0,0 +1,218 @@
+     - PyPSA-Eur supports administrative clustering (NUTS0 to NUTS3), allowing the network to be resolved at highly granular levels. Open-TYNDP uses the TYNDP H2 topology, including Z1 and Z2 setup.
+   * - Gas turbine usage and peaking unit utilisation 
+         - Explore dynamic operational needs of gas turbines given increasing reliance of variable renewable resources
+         - Update assumptions around CH4 to H2 retrofitting projects, which have struggled to compete in current markets
+     - Open TYNDP allows for endogenous retrofitting of CH4 pipeline capacity to H2.
+   * - EV Modelling
+         - Ensure that electricity flows follow charge/discharge cycles of EV batteries
+     - PyPSA-Eur refined Vehicle-to-Grid (V2G) dispatch capacity and temperature-dependent energy demand correction factors for EVs.
+   * - Economic Assessment
+         - incorporate economic assessment of key technologies such as Steam Methane Reformers (SMR), nuclear plants, ammonia regasification terminals which are currently represented without economic attributes
+     - Open TYNDP incorporates default capital investment and fixed and variable operation cost assumptions for all technologies
+   * - Methane pricing structure and formation
+            - Resolve pricing inconsistences between synthetic natural gas, biomethane and hydrogen
+     - TODO
+   * - Ammonia Import costs
+           - Ensure ammonia import costs reflect the entire supply chain
+     - Open TYNDP includes location and capacities of European ammonia plants. Ammonia import prices and volumes can be configured.
+   * - Distinction in Hydrogen usage
+           - Separating hydrogen used directly as a gas from hydrogen used as a feedstock for producing synthetic fuels.
+     - PyPSA-Eur added a suite of technologies for methanol-to-power, reforming, and kerosene, and updated locations/capacities for ammonia plants to accurately distribute demand.
+   * - Flexibility of heat pumps
+           - Incorporate heat-pump modelling into PLEXOS to better represent flexibility, thermal inertia and heat storage
+     - Open TYNDP uses an energy-to-power ratio constraiint for thermal energy storage. Proportional sizing is used for chargers and dischargers.
+       Includes an option to calculate dynamic storage capacities for thermal energy storage. Also includes aquifer thermal energy storage, and supplemental heating such as booster heat pumps.
+   * - Modelling of E-fuels 
+           - Allow transportation of e-methanol, e-methane and e-kerosene via pipelines, ships or tanker trucks
+           - Incorporate full load hours into spatial optimisation of e-fuel refineries and supply infrastructure
+     - <TODO>
+   * - Higher granularity topology
+           - Adding more nodes per country and differentiating between prosumer and non-prosumer households for accurate grid interaction modeling
```

Comment:
that is exaclty what PyPSA-Eur was designed for highly resolved spatial and temporal modelling
- higher spatial resolution is possible, also the data can be added on a higher spatial resolution with finer granularity
- disitribution grid with associated costs and losses can be modelled, but data for existing infrastructure of the distribution grid as well as regional investment costs is missing
- on the distribution level connected are EV charging, rooftop PV, electricity demand for invidivual heating technologies
- different types of PV panels can be modelled on the distribution grid level (various publications about this by Parisa https://www.nature.com/articles/s41467-025-61492-9, https://www.sciencedirect.com/science/article/pii/S0306261924001041